releasenotes/notes/add-layers-rustworkx-core-1caaccb1ca292ca8.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Add a function :meth:`~rustworkx-core.dag_algo.layers` in rustworkx-core to 
+    get the layers of a directed acyclic graph.  This is equivalent to the
+    :func:`.layers` function that existed in the Python API but now exposes it
+    for Rust users too.
```

Comment:
We should have a fix note documenting the behavior change around calling the python function with a graph with cycles.

Reply:
Sure thing!